src/main/scala/zio/prelude/Associative.scala

```diff
@@ -76,102 +158,228 @@ object Associative extends Lawful[AssociativeEqual] {
+    new Commutative[Prod[Long]] with Identity[Prod[Long]] {
+      def combine(l: => Prod[Long], r: => Prod[Long]): Prod[Long] = Prod(l * r)
+      val identity: Prod[Long]                                    = Prod(1)
+    }
 
-  implicit val LongProdAssociative: Associative[Prod[Long]] =
-    make[Prod[Long]]((l, r) => Prod(l * r))
+  /**
+   * The `Commutative` and `Identity` instance for the sum of `Long` values.
+   */
+  implicit val LongSumCommutativeIdentity: Commutative[Sum[Long]] with Identity[Sum[Long]] =
+    new Commutative[Sum[Long]] with Identity[Sum[Long]] {
+      def combine(l: => Sum[Long], r: => Sum[Long]): Sum[Long] = Sum(l + r)
+      val identity: Sum[Long]                                  = Sum(0)
+    }
 
-  implicit val LongSumAssociative: Associative[Sum[Long]] =
-    make[Sum[Long]]((l, r) => Sum(l + r))
+  /**
+   * Derives an `Identity[Map[K, V]]` given an `Associative[V]`.
+   */
+  implicit def MapIdentity[K, V: Associative]: Identity[Map[K, V]] =
+    new Identity[Map[K, V]] {
+      def identity: Map[K, V] = Map()
 
-  implicit def MapAssociative[K, V: Associative]: Associative[Map[K, V]] =
-    make[Map[K, V]]((l, r) =>
-      r.foldLeft(l) {
-        case (map, (k, v)) => map.updated(k, map.get(k).fold(v)(_ <> v))
-      }
-    )
+      def combine(l: => Map[K, V], r: => Map[K, V]): Map[K, V] =
+        r.foldLeft(l) {
+          case (map, (k, v)) => map.updated(k, map.get(k).fold(v)(_ <> v))
+        }
+    }
 
-  implicit def MaxAssociative[A: Ord]: Associative[Max[A]] =
-    make((l: Max[A], r: Max[A]) => if (l >= r) l else r)
+  /**
+   * The `Commutative` instance for the maximum of `A` values for which an
+   * `Ord` is defined.
+   */
+  implicit def MaxCommutative[A: Ord]: Commutative[Max[A]] =
+    Commutative.make((l: Max[A], r: Max[A]) => if (l >= r) l else r)
 
-  implicit def MinAssociative[A: Ord]: Associative[Min[A]] =
-    make((l: Min[A], r: Min[A]) => if (l <= r) l else r)
+  /**
+   * The `Commutative` instance for the minimum of `A` values for which an
+   * `Ord` is defined.
+   */
+  implicit def MinCommutative[A: Ord]: Commutative[Min[A]] =
+    Commutative.make((l: Min[A], r: Min[A]) => if (l <= r) l else r)
 
+  /**
+   * The `Associative` instance for the concatenation of `NonEmptyChunk[A]`
+   * values.
+   */
   implicit def NonEmptyChunkAssociative[A]: Associative[NonEmptyChunk[A]] =
     make(_ ++ _)
 
-  implicit def OptionAssociative[A: Associative]: Associative[Option[A]] =
-    make[Option[A]]((l, r) =>
-      (l, r) match {
-        case (Some(l), Some(r)) => Some(l <> r)
-        case (Some(l), None)    => Some(l)
-        case (None, Some(r))    => Some(r)
-        case _                  => None
-      }
-    )
+  /**
+   * Derives an `Identity[Option[A]]` given an `Associative[A]`.
+   */
+  implicit def OptionIdentity[A: Associative]: Identity[Option[A]] =
+    new Identity[Option[A]] {
```

Comment:
is there a reason not to use `Identity.make`?

Reply:
I think I just copied this from the previous implementation. It looks like we weren't using `make` there. But we definitely can.